src/main/java/by/custom/utilcalculator/directory/BotEntity.java

```diff
@@ -0,0 +1,63 @@
+package by.custom.utilcalculator.directory;
+
+import by.custom.utilcalculator.directory.steps.*;
+
+public class BotEntity {
```

Comment:
Maybe `UserProgress` will be more informative.

Reply:
Done.